docs/DeveloperGuide.adoc

```diff
@@ -574,17 +574,17 @@ can enter my entries incrementally.
 
 *Nice-To-Have*
 
-9. As a user, I can add a module without having to add the code, MCs, or semester taken so that I can quickly calculate CAP without worrying about the modules.
```

Comment:
I just noticed this - I kinda fixed it in a separate PR haha. So maybe you can just revert this.

Reply:
I think can go ahead and merge it, then just resolve the conflict later. Shouldn't be too tough for this one.

Reply:
Ok no problem.